### PR TITLE
Updating Encyclopedia pages

### PR DESCRIPTION
**Pull request by clsackett**
Updating Encyclopedia pages based on requests in Jira tickets: DOC-260 and DOC-282 through DOC-292
1. When there are no references, pages state "N/A" rather than "None" for consistency purposes
2. Some non-external references have the box with arrow symbol since the base URL is not "https://docs.gdc.cancer.gov" 
3. Bulletpoint lists were standardized
- Periods at the end of bulletpoints were removed
- Colons were used instead of dashes

Is it okay to remove links to the Legacy Archive?

### CODE REVIEW

File: docs/Encyclopedia/pages/MAGE-TAB.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ MAGE-TAB archives provide information about the data collection and data process
 
 Contents of a MAGE-TAB file:
 
-* __Investigative Description Format (IDF):__ Provides general information about the study. This includes a brief description, the investigator's contact details, bibliographic references, and a text description of the protocols used in the study<sup>2</sup>.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/MAGE-TAB.md:14]: I see a lot of these are removing the bold from the beginning of each listed item.  Was that requested in one of the tickets?  If it looks better feel free to keep it.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Encyclopedia/pages/Case.md [hunk 2]
```diff
@@ -5,16 +5,16 @@ A case is the collection of all data related to a specific patient in the contex
 
 ## Overview ##
 
-A case refers to a specific cancer patient or cell line in the context of a project<sup>1</sup>. In the GDC Data Model, cases are associated with a project, the samples collected from the case, and the clinical data<sup>2</sup>.
```
[the remark this answers is not in the document]
Comment (clsackett) [reply docs/Encyclopedia/pages/Case.md:8]: Added numbers and closed parentheses to make list more clear

File: docs/Encyclopedia/pages/Clinical_Data.md [hunk 3]
```diff
@@ -17,14 +17,14 @@ In the GDC, clinical data is searchable in the API, Data Portal, or Legacy Archi
...
+1. [GDC Clinical Data Dictionary Entries](/Data_Dictionary/viewer/#?view=table-entity-list&anchor=clinical)
 2. [GDC Clinical Data Harmonization](https://gdc.cancer.gov/about-data/data-harmonization-and-generation/clinical-data-harmonization)
 3. [Selecting Common Cross-Study Clinical Data Elements](https://gdc.cancer.gov/node/777/)
 4. [GDC Data Model](https://gdc.cancer.gov/developers/gdc-data-model)
 
 ## External Links ##
 
-* https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html
-* https://cbiit.nci.nih.gov/
+* [CDE Browser](https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html)
```
[the remark this answers is not in the document]
Comment (clsackett) [reply docs/Encyclopedia/pages/Clinical_Data.md:27]: Done

File: docs/Encyclopedia/pages/MAGE-TAB.md (hunk 1 above)
Comment (clsackett) [reply docs/Encyclopedia/pages/MAGE-TAB.md:14]: Dara had commented on the lack of standardization in how we format lists here: https://jira.opensciencedatacloud.org/browse/DOC-285

I prefer the list items to be bolded, so I went back and fixed them so they should be consistent now

File: docs/Encyclopedia/pages/Case.md [hunk 4]
```diff
@@ -5,16 +5,16 @@ A case is the collection of all data related to a specific patient in the contex
...
+A case refers to a specific cancer patient or cell line in the context of a project<sup>1</sup>. In the GDC Data Model, cases are associated with a project, samples are collected from the case, and the clinical data<sup>2</sup>.
 
 Cases are associated with all clinical data elements as well as the clinical and biospecimen supplemental information<sup>3</sup>. The case entity also serves as an API endpoint which allows for case-level data to be easily retrievable using programmatic methods.
 
 Cases are also the basic unit of registration for the GDC Submission Portal. All cases that are to be submitted to a project need to be registered with dbGaP on a case-level basis prior to submission.  
 
 ## References ##
-1. [GDC Data Dictionary - Case](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=case)
```
[the remark this answers is not in the document]
Comment (clsackett) [reply docs/Encyclopedia/pages/Case.md:15]: Will do

File: docs/Encyclopedia/pages/MAGE-TAB.md [hunk 5]
```diff
@@ -11,13 +11,13 @@ MAGE-TAB archives provide information about the data collection and data process
...
-* __Array Design Format (ADF):__ Defines each array type used. An ADF file describes the design of an array, e.g., which sequence is located at each position on an array and associated annotations<sup>2</sup>.
-* __Description File:__ Provides details about how the data files and molecular material were processed
-* __Changes File:__ Provides details about any changes that were made to each of the files in the MAGE-TAB archive
-* __Manifest File:__ Lists file names and md5sums of the files that should be included in the MAGE-TAB archive
-* __Readme File:__ Provides basic details about the MAGE-TAB archive and the associated study
+* __Investigative Description Format (IDF)__: Provides general information about the study. This includes a brief description, the investigator's contact details, bibliographic references, and a text description of the protocols used in the study<sup>2</sup>
+* Sample and Data Relationship Form (SDRF): Describes the relationships between samples, arrays, data, and other objects used or produced in the study<sup>2</sup>
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/MAGE-TAB.md:15]: I think a bolding is needed here.

**Review by wwysoc2 — APPROVED** [review]
Approved, make the one minor change and feel free to merge into develop.

**Pull request closed without merging**